src/gfn/losses/flow_matching.py

```diff
@@ -122,16 +107,19 @@ def flow_matching_loss(
     def reward_matching_loss(self, terminating_states: DiscreteStates) -> TT[0, float]:
         """Calculates the reward matching loss from the terminating states."""
         assert terminating_states.log_rewards is not None
-        log_edge_flows = self.parametrization.logF(terminating_states)
+        log_edge_flows = self.logF(terminating_states)
         terminating_log_edge_flows = log_edge_flows[:, -1]
         log_rewards = terminating_states.log_rewards
         return (terminating_log_edge_flows - log_rewards).pow(2).mean()
 
     # TODO: should intermediary_states and terminating_states be two input args?
```

Comment:
I would say yes IFF: (they can be decoupled) OR (they don't come from a single function/method call).

In other words, I would find arguments to be more idiomatic in those two situations:

```python
# decoupling
fm_param = FMParametrization(...)
some_states, term_states = some_function()
some_states = modify(some_states)
fm_param.loss(some_states, term_states) # better than fm_param.loss((some_states, term_states))

# multiple funcs
fm_param = FMParametrization(...)
some_states = some_function()
term_states = some_other_function()
fm_param.loss(some_states, term_states) # better than fm_param.loss((some_states, term_states))
```

Reply:
I don't get what you're referring to here

Reply:
I'm also missing some context :)

Reply:
The comment reads "# TODO: should intermediary_states and terminating_states be two input args?" :p